src/streamlink/plugins/okru.py

```diff
@@ -1,14 +1,15 @@
 """
 $description Russian live-streaming and video hosting social platform.
 $url ok.ru
+$url m.ok.ru
```

Comment:
```suggestion
$url m.ok.ru
$url mobile.ok.ru
```

Reply:
This is just for the docs, which show which sites are supported by the plugin. Listing every kind of hostname variation is unnecessary. I updated it to mobile.ok.ru though, to make it more clear that it's the mobile version of the site, and that it's different.